### PR TITLE
Use the prometheus operator to install/run prometheus in our dev cluster

### LINKED ISSUE #89: Use whatever G-Research uses to install prometheus in their clusters for our dev cluster
@kannon92 has more context here, but the main point is contained within the title.

### PR DESCRIPTION
**Pull request by ClifHouck**
Closes #89

### CODE REVIEW

File: config/samples/prometheus.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
```
Comment (kannon92) [inline config/samples/prometheus.yaml:1]: What is this used for?

File: Makefile [hunk 2]
```diff
@@ -254,6 +254,14 @@ helm-install-redis: helm-bitnami
 helm-install-prometheus: helm-bitnami
```
Comment (kannon92) [inline Makefile:254]: Could we remove this as part of this PR?

File: config/samples/prometheus.yaml (hunk 1 above)
Comment (ClifHouck) [reply config/samples/prometheus.yaml:1]: Not sure, this is adapted from: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md#deploying-prometheus

File: Makefile (hunk 2 above)
Comment (ClifHouck) [reply Makefile:254]: Sure

**Review by kannon92 — APPROVED** [review]

**Pull request merged**